Vagrant/bootstrap.sh

```diff
@@ -92,3 +92,38 @@ $RUN_AS_INSTALLUSER git remote add origin "https://github.com/$gitrepo.git"
 $RUN_AS_INSTALLUSER git fetch --all
 $RUN_AS_INSTALLUSER git reset --hard origin/master
 $RUN_AS_INSTALLUSER bundle install
+
+# Setup the application
+
+# 1. Create a migration: rails generate migration CreateDoiRequests
+$RUN_AS_INSTALLUSER bundle exec rails generate migration CreateDoiRequests
+DOI_MIGRATION_FILE=`find db/migrate -type f -name '*_create_doi_requests.rb'|sort|tail -1`
+# 2. Replace the contents of the new migration with this gist: https://gist.github.com/tingtingjh/ab35348f493d565cdcc8
+$RUN_AS_INSTALLUSER cat > $DOI_MIGRATION_FILE <<GIST
+class CreateDoiRequests < ActiveRecord::Migration
+  def change
+    create_table :doi_requests do |t|
+      t.string "collection_id"
+      t.string "ezid_doi", default: "doi:pending", null: false
+      t.string "asset_type", default: "Collection", null: false
+      t.boolean "completed", default: false
+
```

Comment:
I am happy to remove the blank line.  Maybe Tingting should remove it from her Gist?  What is in the script is pasted exactly from her Gist.  In fact, I originally had code that fetched the raw Gist directly from GitHub, but decided to go the simpler, more explicit route of just pasting it directly into the script.